scripts/update-apk.sh

```diff
@@ -35,6 +35,12 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	done
 fi
 
+if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+	for file in app*; do
+		cp $file open-event-dev-${file%%}
+	done
+fi
+
```

Comment:
@iamareebjamal we have renamed the apks generated from deploy branch, thats why i renamed it above

Reply:
You have not appended build flavor in the apk name

Reply:
wouldnt `$file` will add the build flavor like` app-debug` or `app-release`?

Reply:
These are the file names you have written above `open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk ` 

See any problem with these?